profile/README.md

```diff
@@ -0,0 +1,16 @@
+# About Pretendo Network
+Pretendo is an open source Nintendo Network replacement that aims to build custom servers for the Wii U and 3DS family of consoles. Our goal is to preserve the online functionality of these consoles, to allow players to continue to play their favorite Wii U and 3DS games to their fullest capacity.
+
+Visit [our website](https://prentendo.network) for more information on the project.
+
+## Official Links
+
+* [Website](https://prentendo.network)
+* [Discord](https://discord.gg/pretendo)
+* [Forum](https://forum.pretendo.network/)
+* [X/Twitter](https://x.com/pretendonetwork)
```

Comment:
We find "X/Twitter" to be a bit of a mouthful and everyone on the team still refers to it as just Twitter, so we'd prefer to still use the Twitter name and domain (for as long as it works for)